sky/shell/platform/ios/sky_surface.mm

```diff
@@ -112,6 +115,14 @@ - (instancetype)initWithShellView:(sky::shell::ShellView*)shellView {
 
     _shell_view.reset(shellView);
     self.multipleTouchEnabled = YES;
+
+    [[NSNotificationCenter defaultCenter] addObserver:self
```

Comment:
Will do.

sky::SkyEnginePtr has a destructor that frees the ViewportMetrics.  Do we need to do more work for the viewport metrics or does dealloc call the C++ destructors?